cdk/lib/monitoring.ts

```diff
@@ -73,14 +73,14 @@ export class Monitoring extends GuStack {
 
 		const lambdaEventTarget = new LambdaFunction(monitoringLambdaFunction, {
 			event: RuleTargetInput.fromObject({
-				stage: 'PROD', // Both scheduled cmp-monitoring-CODE and cmp-monitoring-PROD are monitoring prod versions
+				stage: stage, // Both scheduled cmp-monitoring-CODE and cmp-monitoring-PROD are monitoring prod versions
 				region: region,
 			}),
 		});
 
 
 		const monitoringDuration: Duration =
-			stage === 'PROD' ? Duration.minutes(prodDurationInMinutes) : Duration.days(1); // Every day for CODE; Every 2 minutes for PROD.
+			stage === 'PROD' ? Duration.minutes(prodDurationInMinutes) : Duration.minutes(prodDurationInMinutes); // Every day for CODE; Every 2 minutes for PROD.
```

Comment:
Do we still want CODE monitoring kicking off every 2 minutes or do we want a `codeDurationInMinutes` instead?

Reply:
No, all the cdk changes will be removed before push to main